src/Components/CreateImageWizard/validators.ts

```diff
@@ -81,3 +83,10 @@ export const getDuplicateMountPoints = (partitions: Partition[]): string[] => {
   }
   return duplicates;
 };
+
+export const isNtpServerValid = (ntpServer: string) => {
+  return (
+    ntpServer !== undefined &&
+    /^[a-z0-9-]+((\.[a-z0-9-]+)?){1,}$/.test(ntpServer)
```

Comment:
what about IPv6? 😇 I wonder what format of IPv6 addresses chrony accepts.

Reply:
The regex can be refined for sure as this is just something I've tried to put together. Will need to do some reading on formats accepted by chrony for NTP servers thought :sweat_smile:

Reply:
Hmm, found something: https://chrony-project.org/doc/3.4/chrony.conf.html, let me try to update the regex a bit...